configs/experiment_schema.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
There is a way in Git to rename a file. So you can apply changes to the existing `announcement_schema.json`, then rename the file, so that the changes will show up properly in the diff (i.e., it may be better to evaluate this as a diff on top of the existing file than a new one).